lib/Board.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class BOARD
+  # This class manages sample and input boards
+  # example_board
+  def example_board
+    b = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
+    b
+  end
+
+  # add_space
+  def add_space(symb, nlen)
+    space = ''
+    (0..nlen - 1).each do
+      space += symb
+    end
+    space
+  end
+
+  # print_board
+  def print_board(arr, clen)
+    ans = ''
+    arr = [[1, 2, 3], [4, 5, 6], [7, 8, 9]] if arr.nil?
+    row = arr.length
+    col = arr[0].length
+
+    cell_length = clen
+    (0..row - 1).each do |i|
+      line = ''
+      if i.positive && i < row
```

Comment:
:bug: Fix this issue, currently after player name and weapon is chosen, the game can not start.

```ruby
****************************************************
##############  Welcome! Tic Tac Toe  ##############
###################  Game Rules  ###################
####  Two players will take turns to mark the  #####
######  spaces on a 3x3 grid. The player who  ######
####  succeeds in placing 3 of their marks in  #####
####  a horizontal, vertical, or diagonal row  #####
#####  wins the game. When there are no more  ######
#####  spaces left to mark, it is consider a  ######
####  draw. To place a mark on the grid, type  #####
#####  the number on the space you would like  #####
######  to mark! As shown below. Good luck!  #######
****************************************************
            What do you wanna be called?            
A
            Choose your Weapon, X or O?             
X
              You are playing against?              
B
            Choose your Weapon, X or O?             
O






Traceback (most recent call last):
        3: from bin/main.rb:89:in `<main>'
        2: from ../tic-tac-toe/Advanced_Building_Blocks/Tic-Tac-Toe/lib/board.rb:28:in `print_board'
        1: from ../tic-tac-toe/Advanced_Building_Blocks/Tic-Tac-Toe/lib/board.rb:28:in `each'
../tic-tac-toe/Advanced_Building_Blocks/Tic-Tac-Toe/lib/board.rb:30:in `block in print_board': undefined method `positive' for 0:Integer (NoMethodError)
Did you mean?  positive?
```